R/util.R

```diff
@@ -471,3 +471,18 @@ unlist_times <- function(x) {
 empty_time <- function() {
   Sys.time()[-1]
 }
+
+
+maybe_unlink <- function(x, recursive = FALSE, dry_run = FALSE) {
+  if (!dry_run) {
+    unlink(x, recursive)
+    return()
+  }
```

Comment:
I know we usually go for a happy path style, but I wonder if this one should be one big if/else?